spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/config/DataFlowClientProperties.java

```diff
@@ -0,0 +1,76 @@
+package org.springframework.cloud.dataflow.rest.client.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.cloud.dataflow.core.DataFlowPropertyKeys;
+
+/**
+ * Configuration properties used in {@link DataFlowClientAutoConfiguration}
+ *
+ * @author Vinicius Carvalho
+ */
+@ConfigurationProperties(prefix = DataFlowPropertyKeys.PREFIX + "client")
+public class DataFlowClientProperties {
+
+	private String target = "http://localhost:9393";
+
+	private Security security = new Security();
+
+	private boolean skipSslValidation = true;
+
+	private boolean enableDsl = false;
+
+	public boolean isEnableDsl() {
+		return enableDsl;
+	}
+
+	public void setEnableDsl(boolean enableDsl) {
+		this.enableDsl = enableDsl;
+	}
+
+	public boolean isSkipSslValidation() {
+		return skipSslValidation;
+	}
+
+	public void setSkipSslValidation(boolean skipSslValidation) {
+		this.skipSslValidation = skipSslValidation;
+	}
+
+	public String getTarget() {
```

Comment:
In a similar discussion on skipper, this property should be renamed to `serverUri`.